registry.go

```diff
@@ -31,38 +31,26 @@ func PathIs(path string) goproxy.ReqConditionFunc {
 	}
 }
 
-var pool *pgx.ConnPool
-
-func afterConnect(conn *pgx.Conn) (err error) {
-	_, err = conn.Prepare("getPackage", `
-		SELECT name, url FROM packages WHERE name = $1
-  `)
-	if err != nil {
-		return
+func getEnv(key, def string) string {
+	k := os.Getenv(key)
+	if k == "" {
+		return def
 	}
-
-	return
+	return k
 }
 
 func main() {
+	memcachedUrl := getEnv("MEMCACHEDCLOUD_SERVERS", "localhost:11211")
 	var err error
-
-	memcached_url := os.Getenv("MEMCACHEDCLOUD_SERVERS")
-	memcached_username := os.Getenv("MEMCACHEDCLOUD_USERNAME")
-	memcached_password := os.Getenv("MEMCACHEDCLOUD_PASSWORD")
-
-	if memcached_url == "" {
-		memcached_url = "localhost:11211"
-	}
-
-	cn, err := mc.Dial("tcp", memcached_url)
+	cn, err = mc.Dial("tcp", memcachedUrl)
 	if err != nil {
 		log.Fatalf("Memcached connection error: %s", err)
 	}
 
-	if memcached_url != "" && memcached_password != "" {
-		err = cn.Auth(memcached_username, memcached_password)
-		if err != nil {
+	memcachedUsername := os.Getenv("MEMCACHEDCLOUD_USERNAME")
+	memcachedPassword := os.Getenv("MEMCACHEDCLOUD_PASSWORD")
+	if memcachedUrl != "" && memcachedPassword != "" {
```

Comment:
`memcachedUrl != ""` is always true because it is provided with a default.  Do you mean `memcachedUsername != ""` ?